configs/nextflow.hopper.config

```diff
@@ -114,11 +118,9 @@ profiles {
     params.distributed = false
     params.intervals = "PH"
     params.bedgz = "PH"
-    params.panelhs = "PH"
-    params.panelhs2 = "PH"
     // RANK MODELS // 
-    params.rank_model_s = "/fs1/resources/scout/rank_models_38/rank_model_v5.2.1_single.ini"
-    params.rank_model = "/fs1/resources/scout/rank_models_38/rank_model_v5.2.1.ini"
+    params.rank_model_s = "/fs1/jakob/resources/grpmax_rank_models/rank_model_v5.2.1_single.ini"
```

Comment:
This must be fixed before finishing up. This is the same rank model, but targeting the CSQ grpmax field instead. I guess we will bunch together all rank model changes somehow (as I think we discussed in the RD meeting)

Reply:
Yes, when everything else is finished

Reply:
Removed now, as genmod no longer is used here